src/Administration/Resources/app/administration/src/module/sw-dashboard/component/sw-dashboard-statistics/sw-dashboard-statistics.scss

```diff
@@ -57,4 +57,8 @@ $content-max-width: 960px;
     &__help-text {
         margin-left: 8px;
     }
+
+    .sw-block-field__block {
```

Comment:
Change is inspired by https://github.com/shopware/shopware/pull/8933/files

Seems like `height: auto` is needed in places where flexbox is used. I'm not sure if we can fix it more globally without breaking other places 🤔 

cc: @jleifeld

Reply:
We wouldn't do global fixes anymore until the QA team is ready with their visual tests. Often times these changes also have side effects somewhere else

Reply:
Alright, so this local fix should be fine then right? 🙂

Reply:
Yeah, correct 🙂